src/main/java/com/myhome/security/WebSecurity.java

```diff
@@ -26,24 +27,29 @@
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
 import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.core.userdetails.UserDetailsService;
 import org.springframework.security.crypto.password.PasswordEncoder;
 
 @Configuration
 @EnableWebSecurity
 public class WebSecurity extends WebSecurityConfigurerAdapter {
   private final Environment environment;
   private final ObjectMapper objectMapper;
-  private final AppUserDetailsService appUserDetailsService;
+  private final UserDetailsService userDetailsService;
   private final PasswordEncoder passwordEncoder;
+  private final UserDetailFetcher userDetailFetcher;
+  private final AppJwtEncoderDecoder appJwtEncoderDecoder;
 
   public WebSecurity(Environment environment,
-      ObjectMapper objectMapper,
-      AppUserDetailsService appUserDetailsService,
-      PasswordEncoder passwordEncoder) {
+      ObjectMapper objectMapper, UserDetailsService userDetailsService,
```

Comment:
RequiresArgsConstructor can be used here

Reply:
Done